src/Core/src/Configuration.php

```diff
@@ -38,6 +40,7 @@ final class Configuration
         self::OPTION_ROLE_ARN => true,
         self::OPTION_WEB_IDENTITY_TOKEN_FILE => true,
         self::OPTION_ROLE_SESSION_NAME => true,
+        self::OPTION_METADATA_SERVICE_TIMEOUT => true,
```

Comment:
missing `OPTION_CONTAINER_CREDENTIALS_RELATIVE_URI`?

Reply:
I wasn't sure about that since that relative uri can't be change. It's different every time an instance is run. That is as far as i know.

Reply:
I got your point, and agreed, people will propably not use it, but for consistency with other options (ie. `SESSION_TOKEN`), I prefer to let users choose the way they want to inject it.

Ie the following code works

```yaml
async_aws:
    config:
        ContainerCredentialsRelativeUri: '%env(CONTAINER_CREDENTIALS_RELATIVE_URI)%'
```

It also let the opportunity for user to share the parameter with sub-system that don't have access to the env variables (docker run, other instane, ...).

Reply:
That makes sense to me. I'll add it